markdown/3.x/en/api/ff-asn.api.md

```diff
@@ -94,5 +94,7 @@ ___
 ### Properties
 | Name | Description |
 | ---- | ----------- |
-| **submit-on-input**&nbsp;(String) **Options**&nbsp;"true",&nbsp;"false" (default: "false") | If this attribute is set to true, the input fields accept values. If set to false, you cannot focus the input fields. |
+| **disable-input-fields**&nbsp;(String) **Options**:&nbsp;"true",&nbsp;"false" (default: "false") | If this attribute is set to true, the input fields accept values. If set to false, you cannot focus the input fields. |
+| **submit-on-input**&nbsp;(String) **Options**:&nbsp;"true",&nbsp;"false" (default: "false") | If this is set to true a filter request is made immediately upon key stroke instead of waiting for ENTER key press. |
+| **submit-delay**&nbsp;(Number)(default: 500)| Time (in milliseconds) that must pass after last keystroke before input is submitted. |
```

Comment:
`submit-delay` wasn't introduced if I recall correctly.

Reply:
It was in the beginning, but we decided to remove it.

Removed from the documentation as well.